reco: get files exposed to specific email address

GalNakash-RecoLabs opened the pull request:
<!-- REMINDER: THIS IS A PUBLIC REPO DO NOT POST HERE SECRETS/SENSITIVE DATA -->
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

maimorag commented on Packs/Reco/Integrations/Reco/Reco.py:
```diff
@@ -1028,6 +1085,64 @@ def get_sensitive_assets_shared_with_public_link(reco_client: RecoClient) -> Com
     )
 
 
+def get_files_exposed_to_email(reco_client: RecoClient, email_account: str) -> CommandResults:
```
```suggestion
def get_files_exposed_to_email_command(reco_client: RecoClient, email_account: str) -> CommandResults:
```

GalNakash-RecoLabs commented:
> 2\. added a suggestion to edit "get_files_exposed_to_email" command so it won't have the same name as the client command, but it's not man

Fixed all issues

maimorag commented:
Hey @GalNakash-RecoLabs,
please address the build error and add the new command to the readme file.

https://app.circleci.com/pipelines/github/demisto/content/359843/workflows/b4514b99-8f1e-4535-a6d0-580f0b575d71/jobs/747049?invite=true#step-119-5388_152
